src/Microsoft.PowerShell.Commands.Utility/commands/utility/XmlCommands.cs

```diff
@@ -679,18 +982,24 @@ private static XmlReader CreateXmlReader(Stream stream)
             return XmlReader.Create(textReader, InternalDeserializer.XmlReaderSettingsForCliXml);
```

Comment:
This seems wrong as a new reader is created every time the property is accessed.